contracts/codecov.yml

```diff
@@ -0,0 +1,12 @@
+comment: false
```

Comment:
hmm we're not using codecov.io as far as I know, maybe we should 🤣 
but for now I think this file is not being used and can be deleted